apps/studio/src/pages/sites/[siteId]/pages/[pageId]/index.tsx

```diff
@@ -179,6 +188,9 @@ const PageSettings = ({
     }
   })
 
+  const fullPermalink =
+    type === ResourceType.RootPage ? "/" : `${prefix}/${permalink}`
```

Comment:
nit: Might be safer (and make more sense) to use `String#join` instead so don't need the conditional.

```suggestion
  const fullPermalink = [prefix, permalink].join('/')
```